File: Hash/Hash01.py
def solution(participant, completion):
    i = 0

    while(i < len(completion)):
        check = completion[i] in participant
        if(check == True):
            participant.remove(completion[i])
            i = i + 1
    return participant[0]

# 정확도 100 효율성 0

File: Hash/Hash01.java
import java.util.HashMap;

class Solution {
    public String solution(String[] participant, String[] completion) {
        String answer = "";
        HashMap<String, Integer> hashMap = new HashMap<String, Integer>();

        for (String e : participant) {
            if (!hashMap.containsKey(e)) {
                hashMap.put(e, 0);
            }
            hashMap.put(e, hashMap.get(e) + 1);
        }

        for (String e : completion) {
            if (hashMap.containsKey(e)) {
                hashMap.put(e, hashMap.get(e) + 1);
            }
        }

        for (String e : participant) {
            if (hashMap.get(e) % 2 != 0) {
                answer = e;
            }
        }
        return answer;
    }
}